dev-db/tarantool/tarantool-9999.ebuild

```diff
@@ -9,8 +9,8 @@ inherit cmake tmpfiles
 MAJORV=$(ver_cut 1)
 MINORV=$(ver_cut 2)
 
-# Ninja makefile generator is only supported in master.
-if [ "${PV}" != 9999 ]; then
+# The Ninja makefile generator has been supported since version 2.10.2.
+if [ "${PV}" != 9999 ] && ver_test ${PV} -le 2.10.0; then
```

Comment:
The comment is about 2.10.2, but the check compares with 2.10.0. It is confusing on the first glance, but then I see `-le` (not `-lt`) and remember that 2.10.1 is yanked. I would try to make it more clear in the comment.

BTW,

```sh
$ . /usr/portage/eclass/eapi7-ver.eclass
$ ver_test 9999 -le 2.10.0 && echo true || echo false
false
```

It seems, we can eliminate the first if-condition.